docs/serviceability.md

```diff
@@ -0,0 +1,76 @@
+# Serviceability
+
+The serviceability document aims to help local application-service developers and SRE engineers to access and service the application-service component. This document will help you understand how to access and understand the application-service logs, how to debug an application-service problem and provides a quick summary on the various questions that you might have regarding application-service. 
+
+## Accessing the Logs
+
+### Deployed Locally
+View the application-service controller logs by running the executable manager. Example,
+
+```
+make install
+make build
+./bin/manager
```

Comment:
This feels like it should go in the deploy doc? Plus I believe some environment variables (disabling webhooks and git token) need to be set before running the binary.

then just say here that the logs are available in the terminal window